src/components/SlateEditor/plugins/toolbar/SlateToolbar.tsx

```diff
@@ -46,10 +46,19 @@ const specialRules: { [key: string]: Partial<Element> } = {
   },
 };
 
-export const toolbarClasses = new BEMHelper({
-  name: 'toolbar',
-  prefix: 'c-',
-});
+const ToolbarContainer = styled.div`
+  border-radius: 4px;
+  position: absolute;
+  z-index: 11;
+  top: -10000px;
+  left: -10000px;
+  margin-top: -6px;
+  opacity: 0;
+  color: black;
+  background-color: white;
+  transition: opacity 0.75s;
+  box-shadow: 3px 3px 5px rgba(153, 153, 153, 0.349019607843137);
```

Comment:
Er litt redd for at hele siden brekker om vi endrer `0.349019607843137` til `0.35`, men jeg foreslår det likevel. Eventuelt sjekke ut om vi bare kan ta noe fra ndla core.